pkg/cloudprovider/aws/instance.go

```diff
@@ -124,6 +129,18 @@ func (p *InstanceProvider) launchInstance(ctx context.Context, provider *v1alpha
 	if err != nil {
 		return nil, fmt.Errorf("getting launch template configs, %w", err)
 	}
+
+	if err = p.checkODFallback(nodeRequest, launchTemplateConfigs); err != nil {
+		logging.FromContext(ctx).Errorf("preparing fleet request, %v", err)
```

Comment:
Don't log in line. We should bubble logs up as part of the fmt.Errorf and log a single time.